### ISSUE TITLE
NormalPeoplePicker is missing input fields once ItemLimit is reached which cause "Required ARIA child role not present: textbox"

### ISSUE THREAD

**Issue by irinatarn** [microsoft/fluentui#14170]
If you have an element that has role="combobox" it is required that it contains or owns a text input element with role textbox or searchbox (see https://www.w3.org/TR/wai-aria-1.1/#combobox)
In NormalPeoplePicker set itemLimit={ 1 }.
Fill it out with a person name.
The input field disappear (div is placed there) but the parent still has role="combobox" which causes accessibility issue.
Here is a codepen but you can repro it in any environment:
https://codepen.io/irinata/pen/ExPrdrw
 I expect the role="combobox" to be removed once the limit of items is reached.

**Comment by paulgildea** [comment]
@smhigley - Do you have a perspective on the right behavior based on the description? Thanks!

**Comment by irinatarn** [comment]
I expect the role="combobox" to be removed once the limit of items is reached.

**Comment by xugao** [comment]
@smhigley - could you advice the right fix here?

**Comment by smhigley** [comment]
This one is interesting because in general I agree -- I expect `role="combobox"` to be removed once the limit is reached, or alternatively to have `aria-readonly` applied.

The interesting part is that there are two other things going on: this particular automated rule will likely be removed soon, since the 1.2 combobox pattern does not have a child text input. The other thing is independent of the specific automated rule and 1.2 pattern, a combobox should not ever have a child list (or listbox, as I think it should be in this case). If the picker list/listbox were moved outside of the element that has `role="combobox"` (or the combobox role is moved somewhere else), then either removing the combobox role or adding readonly would work as solutions. Without that change, the best immediate solution would be to remove the combobox role.

**Comment by xugao** [comment]
To dev to fix, couple options to explore:
1. remove `role=combobox` - this might be the right thing to do. but would it cause confusion for user when same control changes role? 
2. keep `input` inside ComboBox but have it disabled

**Comment by dhanMS** [comment]
Is this issue got released? Can you help in resolving this if it is not yet fixed, as we have accessibility issue open because of this

**Comment by xugao** [comment]
@dhanMS - sorry for the delay. @TristanWatanabe will work on this this week
